### PR TITLE
add WIP chat view

### PR DESCRIPTION
**Pull request by bariod**
some useless format changes snuck into this one as well..

### CODE REVIEW

File: packages/app/src/components/InputBox.vue [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<script setup>
+import { ref } from 'vue'
+
+let input = ref('')
+
+function newMessage() {
+  if (input.value) {
+    this.$emit('sendMessage', input.value) // todo: es5 function has own this context
```
Comment (bariod) [inline packages/app/src/components/InputBox.vue:8]: I'm currently having a problem emitting, as the "this" in this function is not the "global" one.. 
```
Uncaught TypeError: Cannot read properties of undefined (reading '$emit')
```

**Comment by bariod** [comment]
WIP stuff:
- the css is still a mix of your changes and mine :D
- messages still have to be sent to backend
- top bar (with search and user icon) is also not included

File: packages/app/src/components/InputBox.vue [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    input.value = ''
+  }
+}
```
Comment (9j3) [inline packages/app/src/components/InputBox.vue:11]: see https://vuejs.org/guide/components/events.html#usage-with-v-model . Emitting events in vue 3 with the composition api is a bit diffrent in context to vue 2.

File: packages/app/src/views/HomeWrapper.vue [hunk 3]
```diff
@@ -14,247 +16,118 @@ const chatStore = useChatStore()
...
+/**
+ * todo
+ */
+function sendMessage(message) {
+  // todo: send message to backend
+
+  // scrolling still todo
+  // const messageContainer = document.getElementById('message-container')
+  // messageContainer.scrollTop = messageContainer.scrollHeight
+}
```
Comment (9j3) [inline packages/app/src/views/HomeWrapper.vue:29]: this logic should be handlet either in a store action or with the socket.io instance. You can leave it for the moment, but notice this ;)

File: packages/app/src/components/ChatDaySeparator.vue [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<script setup>
+const props = defineProps({
+    dateString: {
+        type: String,
+        required: true
+    }
+})
```
Comment (9j3) [inline packages/app/src/components/ChatDaySeparator.vue:7]: We dont like Dates as a String. Instead, pass dateString as a pure Date object and use the provided date formatting functions of Js: https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString

**Review by 9j3 — CHANGES_REQUESTED** [review]

File: packages/app/src/components/InputBox.vue (hunk 1 above)
Comment (9j3) [reply packages/app/src/components/InputBox.vue:8]: See: https://vuejs.org/guide/components/events.html#usage-with-v-model

**Comment by 9j3** [comment]
@bariod I've done some operational work as announced. This contains switching to pnpm from yarn and fixing prettier/eslint. It now works like a charm 😄

**Comment by 9j3** [comment]
@bariod I also had to pin the eslint module to version 8.22.0 because there were changes to the api of eslint. However, this should not affect you, since you use vscode - the whole thing is a bug of webstorm: https://youtrack.jetbrains.com/issue/WEB-57089/ESLint823-TypeError-thislibOptionsparse-is-not-a-function

File: packages/app/src/components/ChatDaySeparator.vue (hunk 4 above)
Comment (bariod) [reply packages/app/src/components/ChatDaySeparator.vue:7]: Who is "we"? 🌚

Comment (9j3) [reply packages/app/src/components/ChatDaySeparator.vue:7]: The Matrix Agents 🌚

File: packages/app/src/views/HomeWrapper.vue [hunk 5]
```diff
@@ -14,247 +16,118 @@ const chatStore = useChatStore()
...
-                  class="h-6 w-6 stroke-slate-400"
-                  fill="none"
-                  viewBox="0 0 24 24"
-                  stroke="currentColor"
-                  stroke-width="2"
+                  {{ chatUser.fullName }}
+                </p>
+                <p
+                  class="text-xs text-slate-400 -mt-0.5 font-semibold"
+                  x-text="user.email"
```
[the remark this answers is not in the document]
Comment (bariod) [reply packages/app/src/views/HomeWrapper.vue:73]: I don't even know what that was for. It's still a leftover from the initial version.

**Review by 9j3 — APPROVED** [review]

**Pull request merged**